frontend/javascripts/test/puppeteer/dataset_rendering_helpers.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}], no-await-in-loop: 0 */
+/* eslint no-await-in-loop: 0 */
```

Comment:
`0` is the same as false, right?

Reply:
Yes, false doesn't work so 0 seems to be the way to do this